apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,10 @@
+<configuration>
+  <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+      <pattern>%msg%n</pattern>
+    </encoder>
+  </appender>
+  <root level="OFF">
```

Comment:
Is that really what you want for your tests? If you're worried about build output spam, I might instead set Surefire/Failsafe to redirect output to a file.

Reply:
The tests explicitly check for exceptions so I'm not worried about losing the logging output when they run.  ERRORs in the logs during tests add a huge amount of noise and make it look like tests are failing even though the exceptions are expected.  I could switch to redirecting the logs during tests if that's the preferred convention.  I'd rather make that change in the next PR.

Reply:
I don't have a strong opinion either way. Though, from experience, having the log output is often the only way to diagnose why a test failed on a CI system. 🤷

Reply:
Good point.